state/cloudcredentials.go

```diff
@@ -96,6 +100,16 @@ func (st *State) UpdateCloudCredential(tag names.CloudCredentialTag, credential
 			ops = append(ops, updateCloudCredentialOp(tag, credential))
 		} else {
 			ops = append(ops, createCloudCredentialOp(tag, credential))
+			cloudCredKey := cloudCredentialGlobalKey(tag)
+			refcounts, closer := st.db().GetCollection(globalrefcountsC)
+			defer closer()
+
+			refOp, required, err := nsRefcounts.LazyCreateOp(refcounts, cloudCredKey)
```

Comment:
this should be StrictCreateOp, and there should be an upgrade step that initialises the refcounts for existing models/credentials